MANIFEST

```diff
@@ -0,0 +1,37 @@
+# file GENERATED by distutils, do NOT edit
```

Comment:
I do not think the MANIFEST should be added to github. It should be in the .gitignore

Reply:
I have heard diverse opinions, but I shall remove it.